rerun_py/pyproject.toml

```diff
@@ -119,3 +119,11 @@ include = ["rerun_sdk.pth", "rerun_sdk/rerun_demo/colmap_fiat.rrd"]
 locked = true
 name = "rerun_bindings"                                             # name of the .so library that the Python module will import
 python-packages = ["rerun_sdk/rerun", "rerun_sdk/rerun_demo"]
+
+[tool.pytest.ini_options]
+# These conform to Python's Warning Filter syntax:
+# https://docs.python.org/3/library/warnings.html#warning-filter
+filterwarnings = """
+error
+ignore::pytest.PytestUnraisableExceptionWarning
```

Comment:
any more context why this one is ignored?

Reply:
The context is that I've been lazy and discarded this error as what I initially (mis-)read as some internal Pytest stuff. It turns out they were caused by two nasty instance of us not closing files properly, which I now fixed.

Thanks for not letting me be lazy.